src/bench/rollingbloom.cpp

```diff
@@ -28,4 +28,13 @@ static void RollingBloom(benchmark::State& state)
     }
 }
 
+static void RollingBloomReset(benchmark::State& state)
+{
+    CRollingBloomFilter filter(120000, 0.000001);
+    while (state.KeepRunning()) {
+        filter.reset();
+    }
+}
+
 BENCHMARK(RollingBloom, 1500 * 1000);
+BENCHMARK(RollingBloomReset, 150);
```

Comment:
Please adjust this, so that it takes as much time as the other benchmarks (~4seconds, depending on arch)

Reply:
Done.